airflow/www/templates/airflow/variable_edit.html

```diff
@@ -23,7 +23,8 @@
     #val { font-family: monospace; }
   </style>
   <script>
+    const val = document.getElementById('val');
     const height = Math.min(window.innerHeight * 0.5, val.scrollHeight);
-    document.getElementById('val').style.height = `${height}px`;
+    val.style.height = `${height}px`;
```

Comment:
When making a change requested in code review, I momentarily forgot that `height` depends on `val`, so I removed the line where it is defined. It turns out that it still works this way -- apparently `val` is already defined in the global scope by the inherited template -- but it is confusing and brittle, so I felt compelled to make this correction.